tests/unittests/OperatorTest.cpp

```diff
@@ -7025,10 +7025,290 @@ TEST_P(OperatorTest, GroupConvolution) {
+      biasElemQKind, qDim, qStep);
+
+  // Get optimal output TQP based on inspecting the output range for the
+  // particular values of the convolution parameters. If the convolution
+  // sizes are changed than these parameters must be adjusted.
+  auto outputTQP =
+      quantization::chooseQuantizationParams({-6.0, 6.0}, schema, elemQKind);
+  auto *outQTy =
+      mod.uniqueType(elemQKind, outputDims, outputTQP.scale, outputTQP.offset);
+
+  // Prepare parameters for ChannelwiseQuantizedConvolutionNode.
+  Constant *filterCWQ = nullptr;
+  Constant *filterScalesCWQ = nullptr;
+  Constant *filterOffsetsCWQ = nullptr;
+  if (filterFloat) {
+    filterCWQ = filterF;
+  } else {
+    filterCWQ = filterQ;
+    filterScalesCWQ = filterScales;
+    filterOffsetsCWQ = filterOffsets;
+  }
+  Constant *biasCWQ = nullptr;
+  Constant *biasScalesCWQ = nullptr;
+  Constant *biasOffsetsCWQ = nullptr;
+  if (biasFloat) {
+    biasCWQ = biasF;
+  } else {
+    biasCWQ = biasQ;
+  }
+  if (biasScalesExplicit) {
+    biasScalesCWQ = biasScales;
+    biasOffsetsCWQ = biasOffsets;
+  }
+
+  // Create ChannelwiseQuantizedConvolution and Dequantize.
+  ChannelwiseQuantizedConvolutionNode *outQ = F->createChannelwiseQuantizedConv(
+      "CWQConv", inputQ, filterCWQ, biasCWQ, filterScalesCWQ, filterOffsetsCWQ,
+      biasScalesCWQ, biasOffsetsCWQ, outQTy, kernels, strides, pads, group,
+      dilation, schema, elemQKind, biasElemQKind);
+  DequantizeNode *out =
+      F->createDequantize("dequantize", outQ, ElemKind::FloatTy);
+  SaveNode *saveOut = F->createSave("saveOut", out);
+  bindings.allocate(saveOut->getPlaceholder());
+
+  // Create reference floating-point Convolution.
+  auto *refTy = mod.uniqueType(ElemKind::FloatTy, outputDims);
+  ConvolutionNode *ref = F->createConv("Conv", inputF, filterF, biasF, refTy,
+                                       kernels, strides, pads, group, dilation);
+  SaveNode *saveRef = F->createSave("saveRef", ref);
+  bindings.allocate(saveRef->getPlaceholder());
+
+  // Compile and run.
+  EE.compile(CompilationMode::Infer);
+  EE.run(bindings);
+
+  // Extra validations.
+  EXPECT_EQ(F->getNodes().size(), 6);
+  EXPECT_EQ(outQ->getFilter().getElementType(), elemQKind);
+  EXPECT_EQ(outQ->getBias().getElementType(), biasElemQKind);
+
+  // Check error. If bias is carefully quantized then the bias precision does
+  // not matter and so the error tolerance is the same.
+  auto outH = bindings.get(saveOut->getPlaceholder())->getHandle();
+  auto refH = bindings.get(saveRef->getPlaceholder())->getHandle();
+  for (dim_t idx = 0; idx < refH.size(); idx++) {
+    float errVal = std::abs(refH.raw(idx) - outH.raw(idx));
+    EXPECT_TRUE(errVal < 0.05);
+  }
+}
+
+#define TEST_CWQCONV(testName, ...)                                            \
+  TEST_P(OperatorTest, testName) {                                             \
+    ENABLED_BACKENDS("Interpreter", "CPU");                                    \
```

Comment:
nit: Please try to use `CHECK_IF_ENABLED()` on new operator tests.

Reply:
Done.